packs/spell-effects/spell-effect-fiery-body-9th-level.json

```diff
@@ -4,7 +4,7 @@
     "name": "Spell Effect: Fiery Body (9th Level)",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Fiery Body]</p>\n<p>You become living flame, giving you fire immunity, resistance 10 to precision damage, and weakness 5 to cold and to water. Any creature that touches you or damages you with an unarmed attack or non-reach melee weapon takes @Damage[4d6[fire]] damage. Your unarmed attacks deal 2d4 additional fire damage, and your fire spells deal one additional die of fire damage (of the same damage die the spell uses). You can cast @UUID[Compendium.pf2e.spells-srd.Item.Ignition] as an innate spell; the casting is reduced from 2 actions to 1. In fire form, you have a fly Speed of 60 feet and don't need to breathe.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Fiery Body]</p>\n<p>You gain fire immunity, resistance 10 to precision damage, and weakness 5 to cold and to water. Any creature that touches you or damages you with an unarmed attack or non-reach melee weapon takes @Damage[4d6[fire]] damage.</p>\n<p>Your unarmed attacks deal 2d4 additional fire damage, and your fire spells deal one additional die of fire damage (of the same damage die the spell uses).</p>\n<p>In fire form, you have a fly Speed of 60 feet</p>"
```

Comment:
We should delete the heightened effect and add it to our redirected items list (`build/uuid-redirects.json`) since the two effects can be consolidated.

Reply:
How about a followup PR for this? Let this be a simple change and do the unification in the new PR. If you prefer I can keep working on this PR, but I think this fix has more value and can be merged realtively easily, while the rework/unifying will take more time and testing but is mostly maintenance task.

Reply:
That's fine